soundcard/pulseaudio.py

```diff
@@ -46,6 +46,29 @@ def func_with_lock(*args, **kwargs):
         self._pa_operation_unref(operation)
     return func_with_lock
 
+
+def channel_name_map():
+    """
+    Return a dict containing the channel position index for every channel position name string.
+    """
+
+    channel_indices = {
+                          'left': _pa.PA_CHANNEL_POSITION_LEFT,
+                          'right': _pa.PA_CHANNEL_POSITION_RIGHT,
+                          'center': _pa.PA_CHANNEL_POSITION_CENTER,
+                          'subwoofer': _pa.PA_CHANNEL_POSITION_SUBWOOFER} | \
```

Comment:
I just realized that the bitwise-or operator for dictionary unions only became available in 3.9, whereas the rest of soundcard is currently compatible with Python >= 3.5. Would you mind rewriting this to use `.update` instead?

Reply:
Sorry, I just didn't think about that. I fixed the merge and pushed the commit.